bedrock/base/templates/base-resp.html

```diff
@@ -91,7 +98,11 @@
 
     {% block global_nav %}
       {% if LANG.startswith('en-') %}
-        {% include 'includes/global-nav.html' %}
+        {% if show_protocol_nav %}
```

Comment:
Can you include navigation-switch.html here?

Reply:
As this is a base template and we reference the switch multiple times, I've decided to avoid using the include here so that we can use a variable instead (which is afaik a slight perf improvement).